testing_and_setup/compass/ocean/hurricane/USDEQU240at60cr20rr4WD/sandy/config_forward_RK4.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<config case="forward_RK4">
+	<add_link source="../init/ocean.nc" dest="input.nc"/>
+	<add_link source="../init/graph.info" dest="graph.info"/>
+	<add_link source="../init/atmospheric_forcing.nc" dest="atmospheric_forcing.nc"/>
+	<add_link source="../init/points.nc" dest="points.nc"/>
+	<add_link source="../init/forcing.nc" dest="forcing.nc"/>
+
+	<namelist name="namelist.ocean" mode="forward">
+		<template file="forward_template.xml" path_base="script_configuration_dir"/>
+		<template file="forward_wetting_drying_template.xml" path_base="script_configuration_dir"/>
+		<template file="forward_variable_drag.xml" path_base="script_configuration_dir"/>
+		<template file="maxwaterlevel_template.xml" path_base="script_configuration_dir"/>
+
+<!--
+		<option name="config_time_integrator">'split_explicit'</option>
+		<option name="config_dt">'00:00:40'</option>
+		<option name="config_btr_dt">'0000_00:00:04'</option>
+-->
```

Comment:
This could be removed since we're not running SE with WD.

Reply:
I think it helps to keep this in mind, which is why I left it here.